app/views/users/index.html.erb

```diff
@@ -11,4 +12,25 @@
   <% end %>
 </div>
 
+<%########################################################%>
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused codes/comments in order to have a clean codebase.